readthedocs/builds/querysets.py

```diff
@@ -159,13 +175,20 @@ class BuildQuerySet(SettingsOverrideObject):
     _default_class = BuildQuerySetBase
 
 
-class RelatedBuildQuerySetBase(models.QuerySet):
+class RelatedBuildQuerySet(models.QuerySet):
+
+    """
+    For models with association to a project through :py:class:`Build`.
 
-    """For models with association to a project through :py:class:`Build`."""
+    .. note::
+
+       This is currently used for ``BuildCommandViewSet`` from api v2 only.
```

Comment:
```suggestion
       This is only used for ``BuildCommandViewSet`` from api v2.
```

Make the `only` more obvious by starting with it.